corehq/apps/reports/standard/sms.py

```diff
@@ -322,51 +317,121 @@ def uses_locations(self):
         return (toggles.LOCATIONS_IN_REPORTS.enabled(self.domain)
                 and Domain.get_by_name(self.domain).uses_locations)
 
-    @property
-    def rows(self):
-        startdate = json_format_datetime(self.datespan.startdate_utc)
-        enddate = json_format_datetime(self.datespan.enddate_utc)
-        data = SMSLog.by_domain_date(self.domain, startdate, enddate)
-        result = []
-
-        reporting_locations_id = self.get_location_filter() if self.uses_locations else []
-        # Retrieve message log options
+    def _get_rows(self, paginate=True, contact_info=False):
         message_log_options = getattr(settings, "MESSAGE_LOG_OPTIONS", {})
         abbreviated_phone_number_domains = message_log_options.get("abbreviated_phone_number_domains", [])
         abbreviate_phone_number = (self.domain in abbreviated_phone_number_domains)
 
         contact_cache = {}
-        message_type_filter = self.get_message_type_filter()
 
-        for message in data:
-            if message.direction == OUTGOING and not message.processed:
-                continue
+        def filter_by_types(data_):
+            filtered_types = {t.lower() for t in MessageTypeFilter.get_value(self.request, self.domain)}
```

Comment:
I tested out the filters and it's not working because of converting this to lowercase. It looks like we can just remove the `.lower()` call entirely here